tests/test.js

```diff
@@ -1,8 +1,53 @@
 import test from 'tape';
+import data from '../src/fixtures/data';
+import calculator from '../src/lib/calculator';
 
-test('1 = 1 ?', (t) => {
-  const actual = 1;
-  const expected = 1;
-  t.equal(actual, expected, 'Should return 1 when given 1');
+test('Test #0', (t) => {
+  const expected = {
+    liters: 9,
+    cost: 0.0801,
+  };
+
+  const actual = calculator(data, { toilet: 2 });
+  t.deepEqual(actual, expected);
+  t.end();
+});
+
+test('Test #1', (t) => {
+  const expected = {
+    liters: 56,
+    cost: 0.4984,
+  };
+
+  const actual = calculator(data, { shower: 7 });
+  t.deepEqual(actual, expected);
+  t.end();
+});
+
+test('Test #2', (t) => {
+  const expected = {
+    liters: 65,
+    cost: 0.5785,
+  };
+
+  const actual = calculator(data, { toilet: 2, shower: 7 });
+  t.deepEqual(actual, expected);
+  t.end();
+});
+
+test('Test #3', (t) => {
+  const expected = {
+    liters: 107.5,
+    cost: 0.95675,
+  };
+
+  const actual = calculator(data, {
+    toilet: 5,
+    shower: 7,
+    'Faucet tap': 3,
+    'Washing machine': 2,
+    Dishwasher: 1,
```

Comment:
I think this is partly my fault for missing it in a previous PR. But I don't like the inconsistency in this object. Starting with a mixture of upper and lower case, also I personally dislike keys with multiple words with spaces